python/pointmatcher/io.cpp

```diff
@@ -0,0 +1,217 @@
+#include "io.h"
+
+#include "pointmatcher/IO.h"
+
+PYBIND11_MAKE_OPAQUE(PointMatcherIO<ScalarType>::FileInfoVector)
+
+namespace pointmatcher
+{
+	void pybindIO(py::module& p_module)
+	{
+		using PMIO = PointMatcherIO<ScalarType>;
```

Comment:
Does it make sense to make this alias available in the namespace?

Reply:
These aliases are intended to be use only locally and to make the next declaration shorter and clearer, instead of always using something like `PointMatcherIO<ScalarType>::PLYElement::methodName` for each method and member of the class. But, I'm always open to suggestions. 🙂